core/src/main/scala/org/apache/spark/metrics/sink/OpenTsdbSink.scala

```diff
@@ -0,0 +1,165 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.metrics.sink
+
+import java.util.Properties
+import java.util.concurrent.TimeUnit
+
+import com.codahale.metrics.MetricRegistry
+import com.github.sps.metrics.OpenTsdbReporter
+import com.github.sps.metrics.opentsdb.OpenTsdb
+import org.apache.spark.{Logging, SparkEnv}
+import org.apache.spark.metrics.MetricsSystem
+
+/**
+ * Created by kapil on 1/12/15.
+ */
+private[spark] class OpenTsdbSink(val property: Properties, val registry: MetricRegistry,
+                   securityMgr: SecurityManager) extends Sink with Logging {
```

Comment:
Yes, I had fixed it in my local version. But due to inactivity from admins on this pull request, haven't really worked on it. Let me know if you have privileges to merge these changes, I can update the pull request.